ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx

```diff
@@ -194,7 +194,7 @@ const ItemTableContainer = ({
                 </IconButton>
               </Tooltip>
             )}
-            <Tooltip title={t('label.toggle_stash_tab_filter')} placement="bottom">
+            <Tooltip title={t('label.toggle_stash_tab_filter') || ''} placement="bottom">
```

Comment:
See above comment

Reply:
No clue why `TS` was yelling about insufficient types from `Tooltip`. It looked like `t()` is undefined on initialization hence the errors but that shouldn't be true to how we load the translations 🤔